src/cas_cli/data_preparation.py

```diff
@@ -0,0 +1,105 @@
+import typing as t
+import warnings
+
+import anndata
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+
+from cas_cli import exceptions
+
+
+def validate(
+    adata: anndata.AnnData,
+    cas_feature_schema_list: t.List,
```

Comment:
Make an issue for generalization and unification of the concept of CAS schema for the 3 use cases:
- reference data preparation and training curriculum generation
- client-side data validation and sanitation
- pytorch dataloader

Reply:
https://github.com/cellarium-ai/cellarium-cloud/issues/80